src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,103 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+import duke.exception.DukeException;
+
+/**
+ * The class represents an arraylist of tasks.
+ *
+ * @author Bryan Ng Zi Hao
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructor for TaskList.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Constructor for TaskList.
```

Comment:
Could you be more specific what the difference between this and the constructor above is?